### ISSUE TITLE
Drill tool mk4 

### ISSUE THREAD

**Issue by Klaranth** [pandorabox-io/in-game#373]
iceyou 
Perhaps a drill tool "rough drill" could drop lump dust for ore nodes
the amount dropped obviously should be similar to mining the lump and then grinding it

**Comment by SwissalpS** [comment]
I disagree about the amount it would render. A grinding machine is a lot more efficient than a portable drill. Not only in the consistency of the produced dust, but also in the ability to maintain the dust in the machine and not get lost by air-drafts, ambient moisture etc.
